submissions/onyinyeChukwuka/HTML/index.html

```diff
@@ -1,71 +1,117 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Product Page</title>
+    <link href="../css/productpage.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Raleway&display=swap" rel="stylesheet">
 </head>
+
 <body>
     <main>
-        <header>
-            <p> <h1> BOOTSTRAPS </h1></p>
+        <!-- FOR THE HEADER -->
+        <div class="header">
             <nav>
-                <input type="text" placeholder="Search">
-                <input type="Submit" value="Submit">
-                <ul>
-                    <li> <a href="#"> Home </a></li>
-                    <li> <a href="#"> WOMEN</a></li>
-                    <li> <a href="#"> MEN  </a></li>
-                    <li> <a href="#"> Hot Sales </a></li>
-                    <li> <a href="../html-forms/contactform.html" target="_blank"> Contact Us </a></li>
-                    <li> <a href="#"> Sign in </a></li>
-                    <li> <a href="#"> Cart </a></li>
-                </ul>
+                <div class="left-side">
+                    <div class="tab">
+                        <strong>BOOTSTRAPS</strong>
+                    </div>
+                </div>
+                <div class="right-side">
+                    <div class="link-wrapper">
+                        <a href="../html/index.html"> Product Page </a>
```

Comment:
Format this line properly

Reply:
what do you mean @Ireti203

Reply:
Compare line 24 and 27. The spacing between Product Page and the tags. @Onyinye101